rfc/github-cdevents.md

```diff
@@ -0,0 +1,34 @@
+| dev.cdevents.change.created     | pull_request                | This event occurs when there is activity on a pull request. <details><summary>Actions</summary> {"Action type":{"opened"}</br>                                                                                                                                                                                                                                                                                                                         |
+| dev.cdevents.change.updated     | pull_request                | This event occurs when there is activity on a pull request. <details><summary>Actions</summary> {"Action type":{"assigned", "auto_merge_disabled", "auto_merge_enabled", "closed", "converted_to_draft", "demilestoned", "dequeued", "enqueued", <br/>"archived", "labeled", "locked", "milestoned", "ready_for_review","reopened", "review_request_removed", "review_requested", "synchronized", "unassigned", "unlabeled", "unlocked"}</br> |
+| dev.cdevents.change.reviewed    | pull_request_review_comment | This event occurs when there is activity on a pull request review comment. <details><summary>Actions</summary> {"Action type":{"created", "reviewed", "commented"}                                                                                                                                                                                                                                                                                                |
+| dev.cdevents.change.reviewed    | pull_request_review         | This event occurs when there is activity relating to a pull request review. <details><summary>Actions</summary> {"Action type":{"edited", "dismissed", "submitted"}  </details>                                                                                                                                                                                                                                                                                   |
+| dev.cdevents.change.modified    | pull_request_review_thread  | This event occurs when there is activity relating to a comment thread on a pull request. <details><summary>Actions</summary> {"Action type":{"resolved", "unresolved"}  </details>                                                                                                                                                                                                                                                                                |
```

Comment:
I think that this event still belongs to the `dev.cdevents.change.reviewed` case, as the PR itself is not changes, only a review thread is opened or closed.